JakinChan200/CS180_Roald
# Move Form and Decorate Home Page, Add User Context to Countries and Form

## Overview
Adds UserContext that holds username and videos for all countries and home page.
Adds text + entry button to home page with animations.
Moves PubForm from 'Experimental Metrics' to Home page.
Adds additional error reporting to top of country pages.


## Materials
- #36 (does not complete)

## Pictures
![homeaddingdata](https://user-images.githubusercontent.com/43893085/165825597-c24afcaa-8536-42b8-afaf-9391c5467d01.gif)
![image](https://user-images.githubusercontent.com/43893085/165825647-1a2fd6fd-b5a2-4562-a5a1-eeba3d5ee42d.png)
![image](https://user-images.githubusercontent.com/43893085/165825694-4a3ac836-d34e-419c-9541-b2e959e07fb1.png)


## Manual Testing Guide
- Navigate to the home page
- Click the enter data button
- Enter a date without a name and notice that the data was saved
- Navigate to KR, and notice under 'Experimental Metrics' the data is shown
- 

Pull requests should receive approval before merge.
